### PR TITLE
ci: io soak test

### PR DESCRIPTION
**Pull request by blaisedias**
WIP

### CODE REVIEW

File: test/e2e/common/util.go [hunk 1]
```diff
@@ -1084,9 +1094,38 @@ func DeletePools() {
...
+		Version:  "v1alpha1",
+		Resource: "mayastorpools",
+	}
+
+	pools, err := gTestEnv.DynamicClient.Resource(poolGVR).Namespace("mayastor").List(context.TODO(), metav1.ListOptions{})
+	if err != nil {
+		// This function may be called by AfterSuite by uninstall test so listing MSVs may fail correctly
+		logf.Log.Info("DeleteAllPoolFinalisers: list MSPs failed.", "Error", err)
+	}
+	if err == nil && pools != nil && len(pools.Items) != 0 {
```
Comment (chriswldenyer) [inline test/e2e/common/util.go:1109]: nit: you could argue the len check is not needed here

File: test/e2e/common/util.go [hunk 2]
```diff
@@ -1084,9 +1094,38 @@ func DeletePools() {
...
+		for _, pool := range pools.Items {
+			empty := make([]string, 0)
```
Comment (chriswldenyer) [inline test/e2e/common/util.go:1111]: nit: empty could be defined just before it is needed, i.e. just before SetFinalizers()

File: test/e2e/io_soak/io_soak_test.go [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+		},
+			defTimeoutSecs,
+			"1s",
+		).Should(Equal(true))
+	}
+
+	// Execute the application in the pods concurrently.
+	for x, podName := range podNames  {
+		// Run the fio test
+		go runFio(podName, duration, x*500, x*100, doneC, errC)
```
Comment (chriswldenyer) [inline test/e2e/io_soak/io_soak_test.go:145]: Maybe a comment to describe the thinktime parameters could be needed here.

File: test/e2e/common/util.go (hunk 1 above)
Comment (blaisedias) [reply test/e2e/common/util.go]: Done

File: test/e2e/io_soak/io_soak_test.go (hunk 3 above)
Comment (blaisedias) [reply test/e2e/io_soak/io_soak_test.go:150]: Done

File: test/e2e/common/util.go (hunk 2 above)
Comment (blaisedias) [reply test/e2e/common/util.go]: Done

**Pull request closed without merging**